frontend/package.json

```diff
@@ -7,26 +7,36 @@
     "@material-ui/icons": "^4.5.1",
     "@material-ui/lab": "^4.0.0-alpha.31",
     "@material-ui/styles": "^4.7.1",
+    "@react-native-community/async-storage": "^1.7.1",
```

Comment:
This isn't used anywhere, but feel free to ignore if it's needed for something else not specifically imported or something.

Reply:
You're right!